public/modules/custom/grants_applicant_info/src/Element/ApplicantInfoComposite.php

```diff
@@ -37,6 +37,7 @@ public function getInfo(): array {
    * {@inheritdoc}
    *
    * @throws \GuzzleHttp\Exception\GuzzleException
+   * @throws \Drupal\grants_profile\GrantsProfileException|\Drupal\helfi_helsinki_profiili\TokenExpiredException
    */
   public static function getCompositeElements(array $element): array {
```

Comment:
This method shouldn't be static

Reply:
Hmm. This is overriding static getCompositeElements from WebformCompositeBase.

Shouldn't overriding method use same signature as parent?